procrastinate/app.py

```diff
@@ -128,6 +131,13 @@ def _register(self, task: "tasks.Task") -> None:
             )
             self.queues.add(task.queue)
 
+    def _register_builtin_tasks(self) -> None:
+        self.task(
+            functools.partial(builtin_tasks.remove_old_jobs, self.job_store),
+            queue="builtin",
+            name="procrastinate.builtin_tasks.remove_old_jobs",
+        )
```

Comment:
Can we have this in the builtins module and the _register_builtin_tasks() call the fonction in the builtins ? That should make it easier to not forget to add tasks here.

Reply:
I'm not sure I understood what you meant :o I did something, tell me if it's ok with you :>

Reply:
Yep :+1: